pinot-segment-local/src/test/java/org/apache/pinot/segment/local/realtime/converter/RealtimeSegmentConverterTest.java

```diff
@@ -188,6 +188,9 @@ public void testNoRecordsIndexedRowMajorSegmentBuilder()
     assertTrue(segmentMetadata.getAllColumns().containsAll(schema.getColumnNames()));
     assertEquals(segmentMetadata.getStartOffset(), "1");
     assertEquals(segmentMetadata.getEndOffset(), "100");
+    FileUtils.deleteQuietly(tmpDir);
```

Comment:
We may add a `@AfterMethod` method to delete the `TMP_DIR` after every test. All other dirs are sub directory of the root temp dir